frontend/dataset_details.html

```diff
@@ -740,16 +697,16 @@ <h1><!--Section B ---> Questions</h1>
                                 <input type="text" class="form-control dataset_form_select_style" ng-model="objDataset.is_open_licence_txt" placeholder="URL or name of the license">
                             </div>
                         </div>
-                        <div class="col-xs-12 col-sm-2 col-md-2 col-lg-2"></div>
+
                     </div>
 
                     <!-- Question #10 -->
                     <div class="row ">
-                        <div class="col-xs-12 col-sm-2 col-md-2 col-lg-2"></div>
+
                         <div class="col-xs-12 col-sm-8 col-md-8 col-lg-8">
```

Comment:
So here you removed the left and right padding without changing the center column size. Is that the expected outcome?